source/sass/components/_footer-custom-select.scss

```diff
@@ -0,0 +1,36 @@
+.footer-links {
+  select {
+    &.form-control {
+      $background-size: 24px;
+      $padding-x: 12px;
+      $padding-right: $padding-x * 2 + $background-size;
+
+      -moz-appearance: none;
```

Comment:
Note that we're using [autoprefixer](https://github.com/mozilla/donate-wagtail/blob/master/package.json#L6) so there should not be any need to explicitly write vendor-prefixed rules in our SCSS

Reply:
Correct 👍 replaced both with `appearance: none;` (the foundation website may warrant the same update)